create-custom-map-style/README.md

```diff
@@ -0,0 +1,823 @@
+# Create a Custom Map Style with Amazon Location Service
```

Comment:
Generally, if this tutorial is primarily about customizing a style, I don't get to the meat until line 538. By then, there's a good chance I've thrown up my hands dealing with all of the setup. What would you think about restructuring this to start with the Maputnik parts to give the reader a chance to decide if they've created something worth publishing?